com.unity.shadergraph/Editor/GraphUI/Factory/SGModelInspectorFactoryExtensions.cs

```diff
@@ -1,26 +0,0 @@
-//using UnityEditor.GraphToolsFoundation.Overdrive;
```

Comment:
so this file was moved out of /Factory and renamed to ModelInspectorFactoryExtensions.cs, sounds good 👍